src/lib/state/multicall.tsx

```diff
@@ -9,8 +9,15 @@ const multicall = createMulticall()
 
 export default multicall
 
+/**
+ *
+ * @param chainId
+ * @returns The approximate number of blocks written to the corresponding chainId per Ethereum mainnet epoch.
+ */
 function getBlocksPerFetchForChainId(chainId: number | undefined): number {
   switch (chainId) {
+    case ChainId.AVALANCHE:
+      return 30
```

Comment:
@zzmp This aligns with my understanding of the intent of this function, can you confirm?

Reply:
These numbers all seem out of date. Maybe they should be updated in a follow up? It should be appx. how many blocks there are in 12s.

Reply:
I'm googling that it's about 1 block / 2 seconds, which would make this number 6. Is that wrong?

Reply:
@JFrankfurt do you remember the napkin math we did to find 30? Did we find that it was ~2.5tx/s?

Reply:
we were estimating the number of blocks per ethereum block time, so roughly the number of blocks you'd expect avalanche to produce in 12s

Reply:
@cbachmeier after looking closer
1. block time on avalanche is kinda inconsistent. It looks like blocks are produced as work become available, not on a target time spacing.
2. I think I was looking at the network aggregate on avascan, not just the c-chain, when I came up with the initial estimate

Reply:
@zzmp I've added a ticket to revisit the numbers